src/structures/interfaces/TextBasedChannel.js

```diff
@@ -172,88 +172,14 @@ class TextBasedChannel {
   }
 
   /**
-   * Starts a typing indicator in the channel.
-   * @param {number} [count=1] The number of times startTyping should be considered to have been called
-   * @returns {Promise} Resolves once the bot stops typing gracefully, or rejects when an error occurs
+   * Sends a typing indicator in the channel.
```

Comment:
should probably mention how long it lasts

Reply:
How long it lasts is not documented in the [docs](https://discord.com/developers/docs/resources/channel#trigger-typing-indicator).